packages/mdc-tab-indicator/_mixins.scss

```diff
@@ -0,0 +1,76 @@
+/**
+  * @license
+  * Copyright 2018 Google Inc. All Rights Reserved.
+  *
+  * Licensed under the Apache License, Version 2.0 (the "License")
+  * you may not use this file except in compliance with the License.
+  * You may obtain a copy of the License at
+  *
+  *      http://www.apache.org/licenses/LICENSE-2.0
+  *
+  * Unless required by applicable law or agreed to in writing, software
+  * distributed under the License is distributed on an "AS IS" BASIS,
+  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  * See the License for the specific language governing permissions and
+  * limitations under the License.
+*/
+
+@import "@material/theme/mixins";
+
+@mixin mdc-tab-indicator-active {
+  .mdc-tab-indicator {
+    opacity: 1;
+  }
+}
+
+@mixin mdc-tab-indicator-color($color) {
+  .mdc-tab-indicator {
+    @include mdc-tab-indicator-bar-color_($color);
+  }
+
+  .mdc-tab-indicator--icon {
+    @include mdc-tab-indicator-icon-color_($color);
+  }
+}
+
+@mixin mdc-tab-indicator-height($height) {
+  .mdc-tab-indicator {
+    @include mdc-tab-indicator-bar-height_($height);
+  }
+
+  .mdc-tab-indicator--icon {
```

Comment:
I guess this makes sense...you could have a separate mixin for just customizing the height of the icon variant...like `mdc-tab-indicator-icon-height`?

Reply:
Sure, we can make those [mixins](https://github.com/material-components/material-components-web/pull/2461/files/5841b72316af5803ac154d95e1f85f0fddf522b4#diff-7aabd2b6ba3fd8128be6335bf88636eaR73) public.

Reply:
Definitely